src/libraries/Common/src/Interop/Windows/Kernel32/Interop.GetEnvironmentStrings.cs

```diff
@@ -7,7 +7,7 @@ internal static partial class Interop
 {
     internal static partial class Kernel32
     {
-        [DllImport(Libraries.Kernel32, EntryPoint = "GetEnvironmentStringsW", SetLastError = true, CharSet = CharSet.Unicode, BestFitMapping = false, ExactSpelling = true)]
-        internal static extern unsafe char* GetEnvironmentStrings();
+        [DllImport(Libraries.Kernel32, ExactSpelling = true)]
+        internal static extern unsafe char* GetEnvironmentStringsW();
```

Comment:
Has to be also fixed up in https://github.com/dotnet/runtime/blob/master/src/coreclr/src/vm/ecalllist.h#L1055

Reply:
Ah, the pal, thanks.